### ISSUE TITLE
Nextjs bug?

### ISSUE THREAD

**Issue by jurajkrivda** [flsy/react-mapycz#95]
Only test it with simple map example

```
Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
````

**Comment by kolebjak** [comment]
I haven't tried it with NextJS. Could you provide minimal repo?

**Comment by kolebjak** [comment]
Issue should be fixed in version 1.1.1.
